fdbclient/ThreadSafeTransaction.cpp

```diff
@@ -91,6 +91,12 @@ ThreadFuture<Void> ThreadSafeDatabase::createSnapshot(const StringRef& uid, cons
 	return onMainThread([db, snapUID, cmd]() -> Future<Void> { return db->createSnapshot(snapUID, cmd); });
 }
 
+double ThreadSafeDatabase::getMainThreadBusyness() {
+	// Return the main network thread busyness
```

Comment:
Minor nit -- the comment style in this PR is mixed having some above and some below the function signature. I think mostly we're using comments above, so maybe we could switch to that in the few places that are doing the opposite.